packages/bundler/src/modules/Types.ts

```diff
@@ -1,47 +1,34 @@
-import { BigNumberish } from 'ethers'
 import { NotPromise } from '@account-abstraction/utils'
-import { UserOperationStruct } from '@account-abstraction/contracts'
-
-export enum ValidationErrors {
-  InvalidFields = -32602,
-  SimulateValidation = -32500,
-  SimulatePaymasterValidation = -32501,
-  OpcodeValidation = -32502,
-  NotInTimeRange = -32503,
-  Reputation = -32504,
-  InsufficientStake = -32505,
-  UnsupportedSignatureAggregator = -32506,
-  InvalidSignature = -32507,
-}
+import {
+  IEntryPoint__factory,
+  IPaymaster__factory,
+  SenderCreator__factory,
+  UserOperationStruct
+} from '@account-abstraction/contracts'
+import {
+  TestOpcodesAccountFactory__factory,
+  TestOpcodesAccount__factory,
+  TestStorageAccount__factory
+} from '../types'
 
 export enum ExecutionErrors {
   UserOperationReverted = -32521
 }
 
-export interface StakeInfo {
-  addr: string
-  stake: BigNumberish
-  unstakeDelaySec: BigNumberish
-}
-
-export interface ReferencedCodeHashes {
-  // addresses accessed during this user operation
-  addresses: string[]
-
-  // keccak over the code of all referenced addresses
-  hash: string
-}
-
 export type UserOperation = NotPromise<UserOperationStruct>
 
-export interface SlotMap {
-  [slot: string]: string
-}
-
-/**
- * map of storage
- * for each address, either a root hash, or a map of slot:value
- */
-export interface StorageMap {
-  [address: string]: string | SlotMap
-}
+export const abi = Object.values([
```

Comment:
1. don't export generic "abi" from a package.
2. specifically, this one is used when decoding callstack during trace - it should be declared there, not passed through a chain of methods and packages.

Reply:
But this includes ABIs that are only relevant to our project, won't other users want to supply their own ABIs?

Reply:
The test ABIs are not needed, only the "public" ones. They were added for debugging-purposes only, and really can be removed.